README.md

```diff
@@ -5,6 +5,22 @@
 
 ## Get the subtitles:
 ### From Netflix: method 1
+1. You need one of the following web browsers:
+   - [Google Chrome](https://www.google.com/chrome/browser/desktop/)
+   - Firefox
+   - Safari
+   - Microsoft Edge
+   - Opera
+2. Install [Tampermonkey](https://www.tampermonkey.net/), links below:
+   - [Chrome extension](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo)
+   - [Firefox addon](https://addons.mozilla.org/firefox/addon/tampermonkey/)
+   - [Safari howto](https://www.tampermonkey.net/?browser=safari)
+   - [Microsoft Edge addon](https://microsoftedge.microsoft.com/addons/detail/tampermonkey/iikmkjmpaadaobahmlepeloendndfphd)
+   - [Opera addon](https://addons.opera.com/extensions/details/tampermonkey-beta/)
+3. Install [Netflix - subtitle downloader](https://greasyfork.org/en/scripts/26654-netflix-subtitle-downloader) script for Tampermonkey.
+4. To download the subtitles file from Netflix, open the episode in Netflix and download them by clicking on _"Download subs from this episode"_.
```

Comment:
Hi @TasseDeCafe, I based these instructions on your repo https://github.com/TasseDeCafe/netflix-sentence-adder , wdyt? Could you add a screenshot or two as an example please? I don't actually own a Netflix account, so I can't do it myself... Thanks!

Reply:
Hi @isaacbernat, I didn't expect anyone to be interested in this repo I quickly put together 4 years ago, haha. I had initially created it because I didn't know of any good way to extract Netflix subtitles easily for language learning, but the creation of Language Reactor made this more or less unnecessary.

I had planned to revisit this repo because I'm working on my own extension now, so I'll try to answer your question soon, unless you've already figured things out (for some reason I only received a notification for your message today).

Reply:
It's been a while, I am merging this now. But thanks anyway and Happy https://hacktoberfest.com/ !